### PR TITLE
Blocked out tab layout to be used with each tutorial

### LINKED ISSUE #2162: As a user I can see all available tutorials in a tab
Tutorials will be displayed like this:

image

For this issue, we should focus on constructing the reusable component that we'll use for every available tutorial. These components will be rendered inside the tab, and there will be one instance for each available tutorial.

Button functionality will be added as part of a separate issue.

### PR DESCRIPTION
**Pull request by au-shah**
closes #2162 
Signed-off-by: Akshat Shah <akshatshah@akshats-mbp.hursley.uk.ibm.com>

### CODE REVIEW

File: packages/blockchain-ui/src/components/elements/TutorialTile/TutorialTile.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
+@import '@carbon/colors/scss/colors';
+@import '@carbon/layout/scss/_spacing';
+
+.tab-container {
+    display: flex;
+    flex-direction: column;
+    max-width: 70%;
+    background-color: white;
+    margin-bottom: $carbon--spacing-03;
```
Comment (erin-hughes) [inline packages/blockchain-ui/src/components/elements/TutorialTile/TutorialTile.scss:9]: you shouldn't need to do `$carbon--spacing-03` or whatever, just `$spacing-03` should be fine

Comment (au-shah) [reply packages/blockchain-ui/src/components/elements/TutorialTile/TutorialTile.scss]: Done!

**Comment by cazfletch** [comment]
```
          {
                    "command": "reactTutorialGallery.open",
                    "when": "false"
                },
```

This appears twice in the package json. I think i missed it with Erin's pr

**Comment by cazfletch** [comment]
The background doesn't go all the way to the bottom 
image

**Pull request merged**